modules/components/src/Admin/Tabs/Aggregations/TableTab.js

```diff
@@ -26,7 +26,7 @@ export default ({ projectId, graphqlField }) => (
               <strong>Columns order</strong>
               <DraggableListOrderer
                 {...{
-                  itemsList: columnsState.state.columns.map(
+                  itemsList: columnsState.state?.columns.map(
```

Comment:
can you add before map as well just in case?

```
columnsState.state?.columns?.map()
```

Reply:
ok